### DISCUSSION TITLE
Can't open new channels after upgrading to 0.15.1

### DISCUSSION THREAD

**Discussion by pfergi42** [lightningnetwork/lnd#6965] [Troubleshooting] [answered]
### Background

 We recently upgraded from LND 0.14.x --> 0.15.1. Now, when trying to open a new channel, we get this error:

[lncli] rpc error: code = Unknown desc = -5: Invalid address or key: bc1plj9…avc

### Your environment

LND 0.15.1
Ubuntu 20.04
BTCD backend (versionstring": "1.3.0")

### Steps to reproduce

lncli openchannel --sat_per_byte 1 --local_amt <amt> --push_amt 0 <pubkey>

### Expected behaviour

We get a TXID returned.

### Actual behaviour

[lncli] rpc error: code = Unknown desc = -5: Invalid address or key: bc1plj9…avc

Where the address or key changes based on which peer we're trying, but it's always a long string starting with bc1...

**Comment by Roasbeef** [comment, marked as answer]
Make sure you're running btcd v0.23.1.

**Reply by pfergi42** [reply]
Where is the binary we can download for this?

**Reply by pfergi42** [reply]
The instructions to build from source is not working nor is it desirable.

**Reply by guggero** [reply]
Unfortunately there are no release binaries. I created a PR to make building from source easier: https://github.com/btcsuite/btcd/pull/1890
All you need is Golang 1.18 installed and then run `make release-install` on that PR.

**Reply by pfergi42** [reply]
I have upgraded go to 1.19. How do we run 'make release-install' on the PR?

**Reply by pfergi42** [reply]
After running 'git pull', instead of running this command:

GO111MODULE=on go install -v . ./cmd/...

Instead I run this command:

make release-install

I get this error:

make: *** No rule to make target 'release-install'.  Stop.

**Reply by guggero** [reply]
You need to `git checkout` my PR. See https://stackoverflow.com/questions/27567846/how-can-i-check-out-a-github-pull-request-with-git.

**Reply by pfergi42** [reply]
Got it, that worked. Thank you.

**Comment by saubyk** [comment]
Converting this issue to a discussion
